js/kraken.js

```diff
@@ -357,16 +372,29 @@ module.exports = class kraken extends Exchange {
             let id = ids[i];
             let currency = currencies[id];
             // todo: will need to rethink the fees
+            // see: https://support.kraken.com/hc/en-us/articles/201893608-What-are-the-withdrawal-fees-
             // to add support for multiple withdrawal/deposit methods and
             // differentiated fees for each particular method
             let code = this.commonCurrencyCode (currency['altname']);
             let precision = currency['decimals'];
-            result[code] = {
+            let disabledFiatCurrencies = ['CAD', 'USD', 'JPY', 'GBP'];
```

Comment:
It would be great if you moved this array to `'options'` in `describe()` ;)

Reply:
that's done